### PR TITLE
KIALI-1176 Fixes routes to get the right namespace

### PR DESCRIPTION
**Pull request by josejulio**
There were several routes in routes.tsx, and laid out in our VerticalNavigation item.
We can extract data from the routes, e.g. from the route definition `/servicegraph/:namespace` we can extract the value `namespace`.
From the url `/servicegraph/all` we have `namespace=all`, `/servicegraph/default` yields `namespace=default`.

The problem is that we had a route `/servicegraph/all` that matched the url `/servicegraph/all` but didn't had any value for namespace (unlike the route `/servicegraph/:namespace`), routes are matched in order, and `/servicegraph/all` was before `/servicegraph/:namespace`

This namespace value is used [here](https://github.com/kiali/kiali-ui/blob/master/src/pages/ServiceGraph/ServiceGraphRouteHandler.tsx#L59).

Fixed this by adding all the routes to `pathRoutes` and leaving the `routes` as an array for the vertical menu.

### CODE REVIEW

**Comment by lucasponce** [comment]
@josejulio, Probably https://github.com/kiali/kiali-ui/pull/521 affected to this one.

**Comment by josejulio** [comment]
@lucasponce Rebased and tested, the issue was still there before this fix.

@aljesusg please take a look.

**Comment by aljesusg** [comment]
I restarted the travis

**Comment by josejulio** [comment]
> I restarted the travis

I think there is a race condition in one of the tests, I have seen this fail before and eventually it will pass. I'm checking it.

**Comment by josejulio** [comment]
A test does a timeout. 
Increasing the timeout for that test.

File: src/routes.tsx [hunk 1]
```diff
@@ -4,45 +4,42 @@ import IstioConfigPage from './pages/IstioConfigList/IstioConfigListPage';
...
 const baseName = '/console';
 
 /**
  * Return array of objects that describe vertical menu
  * @return {array}
  */
-const routes: Route[] = [
+const routes: MenuItem[] = [
   {
     iconClass: 'fa pficon-topology',
```
[the remark this answers is not in the document]
Comment (josejulio) [reply src/routes.tsx]: +1

File: src/components/Nav/RenderPage.tsx [hunk 2]
```diff
@@ -1,37 +1,27 @@
...
 
     return (
       <PfContainerNavVertical>
         <SwitchErrorBoundary
           fallBackComponent={() => <h2>Sorry, there was a problem. Try a refresh or navigate to a different page.</h2>}
         >
-          {renderRoutes}
           {renderPaths}
           {redirectRoot}
         </SwitchErrorBoundary>
```
[the remark this answers is not in the document]
Comment (josejulio) [reply src/components/Nav/RenderPage.tsx]: +1

**Comment by josejulio** [comment]
@aljesusg did the proposed changes, waiting for travis again

**Pull request merged**